config/src/config/mempool_config.rs

```diff
@@ -11,6 +11,32 @@ use aptos_types::chain_id::ChainId;
 use serde::{Deserialize, Serialize};
 use serde_yaml::Value;
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct LoadBalancingThresholdConfig {
+    /// PFN load balances the traffic to multiple upstream FNs. The PFN calculates the average mempool traffic in TPS received since
+    /// the last peer udpate. If the average received mempool traffic is greater than this threshold, then the below limits are used
+    /// to decide the number of upstream peers to forward the mempool traffic.
+    pub avg_mempool_traffic_threshold_in_tps: u64,
+    /// Suppose the smallest ping latency amongst the connected upstream peers is `x`. If the average received mempool traffic is
+    /// greater than `avg_mempool_traffic_threshold_in_tps`, then the PFN will forward mempool traffic to only those upstream peers
+    /// with ping latency less than `x + latency_slack_between_top_upstream_peers`.
+    pub latency_slack_between_top_upstream_peers: u64,
+    /// If the average received mempool traffic is greater than avg_mempool_traffic_threshold_in_tps, then PFNs will forward to at most
+    /// `max_number_of_upstream_peers` upstream FNs.
+    pub max_number_of_upstream_peers: u8,
+}
+
+impl Default for LoadBalancingThresholdConfig {
+    fn default() -> LoadBalancingThresholdConfig {
+        LoadBalancingThresholdConfig {
+            avg_mempool_traffic_threshold_in_tps: 0,
+            latency_slack_between_top_upstream_peers: 10,
```

Comment:
Is 10 ms sufficient? Even with enable_max_load_balancing_at_any_load=true this could be the limiting factor? Or are you thinking of overriding this when deployed to PFNs?

Reply:
I also set `max_number_of_upstream_peers: 1` in the next line. So, we'll be using only the top 1 peer (no load balancing). In this setting, `latency_slack_between_top_upstream_peers` doesn't matter.
We start load balancing only after 500 TPS workload.

Reply:
So in the case we would want load balancing without having to see increased load, the configuration to do is
1. Increase this value (e.g., 50ms?)
2. Set enable_max_load_balancing_at_any_load = true

Is my understanding correct?

Reply:
Yes. That sounds accurate.